test/nut/open.nut.ts

```diff
@@ -65,6 +66,15 @@ describe('test org:open command', () => {
     expect(result.url).to.include('/visualEditor/appBuilder.app?pageId');
   });
 
+  it('should produce the URL for an existing flow', () => {
+    const result = execCmd<OrgOpenOutput>(`force:source:open --source-file ${flowPath} --url-only --json`, {
```

Comment:
any reason to be using the deprecated command here?

Reply:
nope, just that the others were.  A mix is probably best.